lib/galaxy/schema/groups.py

```diff
@@ -0,0 +1,95 @@
+from typing import List
+
+from pydantic import (
+    Field,
+    Required,
+)
+
+from galaxy.schema.fields import EncodedDatabaseIdField
+from galaxy.schema.schema import Model
+
+
+class GroupIndexResponse(Model):
+    """Response schema for a group."""
+
+    model_class: str = Field(
```

Comment:
The `model_class` here is not just a string, is a particular literal and must always be the same. Check out https://github.com/galaxyproject/galaxy/blob/05fe2868f3a8af1b634579f5c3ca2b47d538e357/lib/galaxy/schema/schema.py#L431 you can also find that there are existing models that you can reuse for groups. You can move them to /galaxy/schema/groups.py` if you want.